app/scripts/services/hpa.js

```diff
@@ -146,15 +156,19 @@ angular.module("openshiftConsole")
         return $q.when([]);
       }
       return MetricsService.isAvailable().then(function(metricsAvailable) {
+        var isV2HPA = hasV2HPAAnnotations(hpaResources);
```

Comment:
So looking at this again, it will skip CPU request validation if any of the resources uses v2 metrics. So if you have two:

1. v2 HPA
2. v1 HPA whose target has no CPU request

The CPU request warning will be dropped. Maybe that's OK since you really shouldn't have two HPA for the same resource, but pointing it out.

Reply:
Noting our discussion in case I come back to read these comments later, having 2 HPA is a problem that is more severe && MUST be fixed.  The CPU request warning might be a problem but is subordinate.